docs/template-bugReport.md

```diff
@@ -0,0 +1,15 @@
+**Bug Report**
+
+
+**Environment**
+**Type (dev/staging/live):**
+**Version or Branch, Commit Hash:**
+
```

Comment:
For this section only, the explanation may be replaced with examples. Consider the following alternative:
```markdown
**Environment (dev/staging/live, version)**

e.g. live server at V5.93
e.g. `master` branch at commit 1234567

```